Scripts/generate.sh

```
@@ -25,6 +31,30 @@ contains() {
+rename_generated_filename() {
+  local old="$1"
+  local new="$2"
+  local old_path="$OUTPUT_DIR_CHAT/models/${old}.swift"
+  local new_path="$OUTPUT_DIR_CHAT/models/${new}.swift"
+  [[ -f "$old_path" ]] && mv "$old_path" "$new_path"
+}
+
+rename_generated_type() {
+  local old="$1"
+  local new="$2"
+  find "$OUTPUT_DIR_CHAT" -name '*.swift' -exec sed -i '' -E "s/[[:<:]]$old[[:>:]]/$new/g" {} +
```

Shellcheck (0.11.0)

[error] Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).

(SC1087)